templates/nextcloud.nomad

```diff
@@ -0,0 +1,80 @@
+job "nextcloud" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "nc" {
+    task "nextcloud" {
+      driver = "docker"
+      config {
+        image = "liquidinvestigations/liquid-nextcloud"
+        volumes = [
+          "${liquid_volumes}/nextcloud/nextcloud:/var/www/html",
+          "${liquid_collections}/ncsync/data:/var/www/html/data/ncsync/files",
+        ]
+        args = ["/bin/sh", "-c", "chown -R 33:33 /var/www/html/ && echo chown done && /entrypoint.sh apache2-foreground"]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "nextcloud"
+        }
+      }
+      template {
+        data = <<EOF
+            NEXTCLOUD_POSTGRES_HOST =
+              {{- range service "nextcloud-pg" -}}
+                {{ .Address }}:{{ .Port }}
+              {{- end }}  
```

Comment:
I've changed this pattern in the other templates because it's dangerous. It allows the env var to be set to weird values. This is somewhat safer:
```
{{- range service "nextcloud-pg" }}
  NEXTCLOUD_POSTGRES_HOST = {{.Address}}:{{.Port}}
{{- end }}
```

Notice how only the open braces have whitespace removal (`{{-`) while the closing braces don't (`}}`) - if we stick to this rule we avoid bugs with collapsed newlines that result in concatenated env vars.